### PR TITLE
Migrate readme examples to v5

### PR DESCRIPTION
**Pull request by SplotyCode**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,18 +29,16 @@ $pheanstalk = Pheanstalk::create('127.0.0.1');
...
-
-$pheanstalk
-    ->useTube($tube)
-    ->put(
-        json_encode(['test' => 'data']),  // encode data in payload
-        Pheanstalk::DEFAULT_PRIORITY,     // default priority
-        30, // delay by 30s
-        60  // beanstalk will retry job after 60s
-     );
+$pheanstalk->useTube($tube)
```
Comment (BBrunekreeft) [inline README.md:32]:
```suggestion
$pheanstalk->useTube($tube);
```

File: README.md [hunk 2]
```diff
@@ -29,18 +29,16 @@ $pheanstalk = Pheanstalk::create('127.0.0.1');
...
+$pheanstalk>put("job payload goes here\n");
+
+$pheanstalk->useTube($tube)
```
Comment (BBrunekreeft) [inline README.md:35]:
```suggestion
$pheanstalk->useTube($tube);
```

**Review by BBrunekreeft — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -29,18 +29,16 @@ $pheanstalk = Pheanstalk::create('127.0.0.1');
...
+$pheanstalk-->put(
+    json_encode(['test' => 'data']),  // encode data in payload
+    Pheanstalk::DEFAULT_PRIORITY,     // default priority
+    30, // delay by 30s
+    60  // beanstalk will retry job after 60s
+);
```
Comment (SamMousa) [inline README.md:41]: Use named params, they are more readable and don't require comments!

**Comment by SamMousa** [comment]
Some minor things, thanks in advance!!

**Review by SamMousa — CHANGES_REQUESTED** [review]

**Review by SamMousa — APPROVED** [review]
Thanks!

**Pull request merged**